recipes/naturalneighbor/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+python:
```

Comment:
You don't need this file. If you have to skip Python <3.12 you can do it in the recipe.

Reply:
I tried constraining the python version to <3.12 in the recipe but then I got a linting error:
> Hi! This is the friendly automated conda-forge-linting service.
> 
> I wanted to let you know that I linted all conda-recipes in your PR (`recipes/naturalneighbor/meta.yaml`) and found some lint.
> 
> Here's what I've got...
> 
> For **recipes/naturalneighbor/meta.yaml**:
> 
>     * ❌ Non noarch packages should have python requirement without any version constraints.
> 
>     * ❌ Non noarch packages should have python requirement without any version constraints.
> 
> 
> This message was generated by GitHub Actions workflow run https://github.com/conda-forge/conda-forge-webservices/actions/runs/12433437281. Examine the logs at this URL for more detail.

However adding supported python versions in the conda_build_config.yaml worked.

Reply:
Please remove this file and follow the review. I can't see the logs with all the extra commits.